listenbrainz/db/dump_manager.py

```diff
@@ -157,12 +169,23 @@ def create_incremental(location, threads, dump_id):
             write_hashes(dump_path)
         except IOError as e:
             current_app.logger.error('Unable to create hash files! Error: %s', str(e), exc_info=True)
-            return
+            sys.exit(-1)
+
+        try:
+            if not sanity_check_dumps(dump_path, 6):
+                return sys.exit(-1)
+        except IOError as e:
```

Comment:
`os.listdir()` raises `OSError` (in fact, all `os.` methods do), think about changing those, those are aliases now, but they may not in the future.